api/hypershift/v1beta1/hostedcluster_types.go

```diff
@@ -1248,9 +1248,78 @@ const (
 // +kubebuilder:validation:Enum=Karpenter
 type Provisioner string
 
+// Configures when and how to scale down cluster nodes.
+type ScaleDownConfig struct {
+	// enabled determines whether the Cluster Autoscaler should scaled down this cluster.
+	//
+	// +kubebuilder:default=Enabled
+	// +kubebuilder:validation:Enum=Enabled;Disabled
+	// +optional
+	Enabled string `json:"enabled,omitempty"`
+
+	// delayAfterAddSeconds sets how long after scale up the scale down evaluation resumes in seconds
```

Comment:
The limits should be documented within the godoc, users won't have access to the validations as they are written here

The default value should also be added in the doc

```
// It must be between 0 and 10000.
// When omitted, this means no opinion, and the platform is left to choose a reasonable default.
// The default value is 60.
```

Reply:
done